### ISSUE TITLE
[Accessibility] Keyboard Inaccessibility of "Company-info" Section (Non-Focusable)

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4961]
## Describe the bug
- The "Company-info" section in the top left of the interface is not keyboard-focusable. This makes it difficult for keyboard-only users to determine if they are currently positioned within that specific section of the UI, reducing their awareness of their location within the Ushahidi Platform

## To Reproduce
1. Launch the local host
2. Attempt to focus on the "Company-info" section using only the keyboard.
3. Observe that the "Company-info" section cannot be focused on or highlighted using the keyboard.

## Expected behavior
The "Company-info" section should be fully keyboard-accessible, allowing users to navigate to and focus on that part of the UI using only the keyboard. This would improve the our platform's usability for keyboard-only and assistive technology users.

## Screenshots
[Screencast from 19-06-2024 12:02:41 ASUBUHI.webm](https://github.com/ushahidi/platform-client-mzima/assets/124133577/77c9d3e5-74bc-4eb9-a45a-af7cbe862705)
